entrypoint.sh

```diff
@@ -30,6 +30,10 @@ if [ "$INPUT_CONTAINER_PUSH_ENABLED" = 'true' ]; then
   echo "End: Publish Image to ECR"
 fi
 
+echo "Start: Trivy Scan"
```

Comment:
This should be run before docker push is done. It makes **0** sense to do it after docker push.

Reply:
Refactored to run trivy push and then docker push.